lib/integer.gd

```diff
@@ -990,6 +990,7 @@ DeclareGlobalFunction( "SmallestRootInt" );
 ##  <Description>
 ##  prints the prime factorization of the integer <A>n</A> in human-readable
 ##  form.
+##  See also <Ref Func="StringPP"/>.
```

Comment:
Hm, I didn't even know that this exists. It's not documented in GAP 4.8. The name is not illuminating, but I see now that it is used in CTblLib and LOOPS  so perhaps renaming it to `StringFactorsInt` is not an option.  

What about introducing `StringFactorsInt` and then declaring synonym `StringPP` and then eventually making it obsolete?